Lib/test/test_ast.py

```diff
@@ -243,6 +243,13 @@ def to_tuple(t):
 
 class AST_Tests(unittest.TestCase):
 
+    def _is_ast_node(self, name, node):
+        if not isinstance(node, type):
+            return False
+        if "ast" not in node.__module__:
+            return False
+        return isinstance(node, type) and name != 'AST' and name[0].isupper()
```

Comment:
You don't need here the `isinstance(node, type)` because if is false it would have exited in the first branch, no?

Reply:
right, I forgot to remove it.